content/en/tools/ledgergraph/ledgergraph-index.md

```diff
@@ -148,19 +148,46 @@ You should now see a list of flows printed to the console, including those liste
 
 ## Configuration Parameters
 
-You can tune **LedgerGraph**'s behaviour through a small set of configuration parameters, shown in this table:
+You can tune **LedgerGraph**'s behaviour through the following configuration parameters:
 
-{{< table >}}
-|Configuration Parameter|Default Value|Acceptable Value(s)|Description|
-|-|:-:|:-:|-|
-|`transactionReaderPageSize` &dagger; |`100`|`10` to `10,000,000`|The number of transactions to include in the result set when querying the database during graph initialization.|
-|`transactionReaderPoolSize` &Dagger;|`10`|`1` to `1000`|The number of threads to use when deserializing transaction data during graph initialization.|
-|`onDemand`|`true`| `true`, `false`|When set to `true`, your LedgerGraph becomes an on-demand service, active only when triggered by the [Archive Service](../../../en/platform/corda/4.9/enterprise/node/archiving/archiving-setup.md). This saves heap memory usage.
-{{< /table >}}
+ 
+
+
+### `transactionReaderPageSize`
+
+**Default value:** 100
+
+**Possible values:** 10 to 10000000
+
+**Description:** The number of transactions to include in the result set when querying the database during graph initialization.|
+
+{{< note >}} Because there can be an extremely large number of transactions in a node's vault, it is important to select an appropriate page size for your database to optimize retrieval performance. Some amount of experimentation may be required on your part to find/define the best value to be used here, so we don't recommend the default value for most production environments. {{</ note >}}
```

Comment:
```suggestion
{{< note >}} Because there can be an extremely large number of transactions in a node's vault, it is important to select an appropriate page size for your database to optimize retrieval performance. Some amount of experimentation may be required on your part to find/define the best value to be used here, so we do not recommend the default value for most production environments. {{</ note >}}
```